packages/data-explorer/src/lib/api.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
+  baseURL: '/',
+  timeout: 1000,
+  validateStatus: function (status) {
+    if (status === 401) {
+      window.location.href = '/login'
+    }
+    return true
```

Comment:
What happens to the original validateStatus code ? 
`return status >= 200 && status < 300; // defaull`
I think we would need both

Reply:
Added default behaviour back in